container/src/main/java/com/flipkart/poseidon/filters/HystrixContextFilter.java

```diff
@@ -70,7 +71,9 @@ private void ingestResponseBasedMetrics(ServletResponse response) {
         // Ingest API response status codes for HttpServletResponse
         if (response instanceof HttpServletResponse && !StringUtils.isNullOrEmpty(RequestContext.get(ENDPOINT_NAME))) {
             String status = (((HttpServletResponse) response).getStatus() / 100) + "XX";
-            Metrics.getRegistry().counter("poseidon.api." + RequestContext.get(ENDPOINT_NAME) + "." + status).inc();
+            Metrics.getRegistry()
+                    .counter("poseidon.api." + RequestContext.get(ENDPOINT_NAME) + "_" + RequestContext.get(RequestConstants.METHOD) + "." + status)
```

Comment:
We use "." as separator timer (in APILegoSet.java). It would be good to move this to a util function.